lib/screens/play/play_page.dart

```diff
@@ -25,6 +27,28 @@ class PlayPage extends StatelessWidget {
         child: Scaffold(
             appBar: AppBar(
               title: Text(title),
+              actions: <Widget>[
+                IconButton(
+                  icon: Icon(Icons.help),
+                  tooltip: 'Quick Help',
+                  onPressed: () {
+                    Navigator.push(
+                      context,
+                      MaterialPageRoute(builder: (context) => QuickRulesPage()),
+                    );
+                  },
+                ),
+                IconButton(
+                  icon: Icon(Icons.settings),
+                  tooltip: 'Settings',
+                  onPressed: () {
+                    Navigator.push(
+                      context,
+                      MaterialPageRoute(builder: (context) => SettingsPage()),
+                    );
+                  },
+                ),
+              ],
```

Comment:
Would be ace to have widget tests for the two of these - something like this for each?:

```
  testWidgets('should launch the quick rules page', (WidgetTester tester) async {
    final state = makeGameState();
    state.nextCard();
    await tester.launchWidget(child: PlayPage('New Game', state));

    await tester.tap(find.byIcon(Icons.settings));
    await tester.pumpAndSettle();

    final settingsFinder = find.byType(SettingsPage);
    final playPageFinder = find.byType(PlayPage);
    expect(settingsFinder, findsOneWidget);
    expect(playPageFinder, findsNothing);

    await tester.pageBack();
    await tester.pumpAndSettle();
    expect(settingsFinder, findsNothing);
    expect(playPageFinder, findsOneWidget);
  });
```